specs/agents/log-correlation.md

```diff
@@ -9,7 +9,33 @@ By adding trace context to log records, users will be able to move between the A
 Logging frameworks and libraries may provide an MDC (Message Diagnostic Context) that allow to also
 reuse those fields in log message formats (for example in plain text).
 
-Log correlation is implemented by adding the following fields to a log document:
+Log correlation relies on two sets of fields:
+- [metadata fields](#metadata-fields)
+  - They allow to build the per-service logs view in UI.
+  - They are implicitly provided when using log-sending by the agent metadata.
+  - When using ECS logging, they might be set by the application.
+- [per-log-event fields](#per-log-event-fields): `trace.id`, `transaction.id` and `error.id`
+  - They allow to build the per-trace/transaction/error logs view in UI.
+  - They are added to the MDC
```

Comment:
I'd avoid the rather Java/slf4j-specific term MDC. Maybe just say that they are injected to individual log events?